config/jobs/kubernetes/sig-cloud-provider/azure/generate.sh

```diff
@@ -164,7 +169,7 @@ $(generate_presubmit_annotations ${branch} pull-kubernetes-e2e-capz-azure-disk-v
     extra_refs:
       - org: kubernetes-sigs
         repo: cluster-api-provider-azure
-        base_ref: release-0.5
+        base_ref: release-1.0
```

Comment:
should we separate the commit for switching the capz release to 1.0 so we can easily revert in case things we go wrong?

Reply:
sure that makes sense

Reply:
once this merges I will open a second to move to 1.0